manageServices/serviceManager.py

```diff
@@ -31,6 +32,8 @@ def managePDNS(self):
         data = ProcessUtilities.outputExecutioner('sudo cat ' + path).splitlines()
         #data = subprocess.check_output(shlex.split('sudo cat ' + path)).decode("utf-8").splitlines()
 
+        path = os.system("chmod 666 /etc/powerdns/pdns.conf")
```

Comment:
@erselbey can't make this file word-readable as it contains sensitive information.

Reply:
That file sometimes gives an error and access is blocked. That's why I set it up.

Reply:
That is a security concern if this file becomes word-readable.